core/client/fs/src/main/java/alluxio/client/block/stream/GrpcBlockingStream.java

```diff
@@ -124,8 +124,9 @@ public void send(ReqT request, long timeoutMs) throws IOException {
    */
   public void send(ReqT request) throws IOException {
     if (mClosed || mCanceled || mClosedFromRemote) {
-      throw new CancelledException(formatErrorMessage(
-          "Failed to send request %s: stream is already closed or canceled.", request));
+      LOG.debug("Failed to send request {}: stream is already closed or canceled. ({})",
```

Comment:
(nit) can you update the java doc/signature to indicate that it is sent best effort?

Reply:
Updated, thanks.